src/components/ResourceBrowser/ResourceList/ConnectingToClusterState.tsx

```diff
@@ -89,7 +84,8 @@ export default function ConnectingToClusterState({
     }
 
     const renderSelectionState = () => {
-        if (loader && !errorMsg) {
+        /* NOTE: should show loading irrespective of errorMsg */
+        if (loader) {
```

Comment:
loader should stop in case of error

Reply:
Since I am using useAsync in k8sResourceTabComponent, the error message is retained until the callback re-runs and finishes. When I use the reload function returned by useAsync, the loading variable from useAsync has a transition between false->true->false, but that is not reflected on the screen since the error message is retained.

Additionally ConnectingToCluster... is used only in 2 places - ClusterOverview and K8sResourceTabComponent. So what would you prefer? Keep the old check (loader && !errorMsg) or make sure this does not arise in ClusterOverview?